terraform/config_production.tfvars

```diff
@@ -0,0 +1,8 @@
+region        = "eu-west-2"
+environment   = "production"
+base_domain   = "ott-production.co.uk"
+cpu           = 512
```

Comment:
Honestly not sure how to judge this one (the duty calculator doesn't do lots and lots of heavy computation or fragment caching). It feels like we're going to want to trim the production one:
```
Showing health and status for app tariff-duty-calculator-production in org trade-tariff / space production as william.fish@enginegroup.com...
name:              tariff-duty-calculator-production
requested state:   started
routes:            tariff-duty-calculator-production.london.cloudapps.digital, www.trade-tariff.service.gov.uk/duty-calculator
last uploaded:     Tue 29 Aug 11:33:14 BST 2023
stack:
docker image:      777015734912.dkr.ecr.eu-west-2.amazonaws.com/tariff-dutycalculator:release-20230829-1031

type:           web
sidecars:
instances:      8/8
memory usage:   8192M
     state     since                  cpu    memory         disk           details
#0   running   2023-08-29T10:34:37Z   0.2%   416.2M of 8G   144.4M of 1G
#1   running   2023-08-29T10:34:37Z   0.2%   417.8M of 8G   144.4M of 1G
#2   running   2023-08-29T10:34:31Z   0.2%   420M of 8G     144.4M of 1G
#3   running   2023-08-29T10:34:41Z   0.1%   421.5M of 8G   144.4M of 1G
#4   running   2023-08-29T10:34:19Z   0.2%   403.3M of 8G   144.4M of 1G
#5   running   2023-08-29T10:34:25Z   0.2%   411.1M of 8G   144.4M of 1G
#6   running   2023-08-29T10:34:19Z   0.2%   419.5M of 8G   144.1M of 1G
#7   running   2023-08-29T10:34:27Z   0.2%   430.5M of 8G   144.4M of 1G
```

Reply:
I'd guess that 8/8 of these never hit cpu/memory constraints

Reply:
Makes sense to me! We can knock it down, the smallest we can do is:

```
cpu    = 256
memory = 512
```

Which corresponds to .25 of a [vCPU](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_size) and 512M.

That's probably a little tight given those tasks are hovering around 400-430M, but what do you think? We can have 1024M for the task alongside .25 vCPU.